### ISSUE TITLE
Train to generate about Business

### ISSUE THREAD

**Issue by MrMegamind** [minimaxir/textgenrnn#253]
Hello Experts
I tried to modify this for a small assignment I have but could not figure out.
I can pay if someone expert can guide me through the process.

The requirement is I provide some text about the a business it will include about, a paragraph about services and any recent news.

It should output a unique paragraph stating something like personalized research on a business

Thanks

**Comment by noob10293** [comment]
I don't think this is the kind of text generation you're looking for. it basically just takes input text, and generates more text in the style of the input text. tl;dr if you give it a bunch of information about your business, it'll just give you the same info
eg:
you put:
business name: abc.corp
provides abcs, deft, anteaters, and flying monkeys.

it gives:
provides name: flying monkeys.abcs
business deft, corp, abc, anteaters

as you can see, it's mostly suited for creative work such as generating stories or plays, not really practical usage.

**Comment by MrMegamind** [comment]
Thanks a lot for writing on this, can you guide me the right route.
